pom.xml

```diff
@@ -54,20 +54,23 @@ under the License.
     <modules>
         <module>flink-table-store-core</module>
         <module>flink-table-store-connector</module>
+        <module>flink-table-store-kafka</module>
     </modules>
 
     <properties>
         <flink.version>1.15-SNAPSHOT</flink.version>
         <scala.binary.version>2.12</scala.binary.version>
         <slf4j.version>1.7.15</slf4j.version>
         <log4j.version>2.17.1</log4j.version>
+        <junit4.version>4.13.2</junit4.version>
```

Comment:
Is there any legacy reason for us to both keep junit4 and junit5?

Reply:
Flink ITCase base class still use junit4